app/views/claims/student_loan_amount.html.erb

```diff
@@ -3,21 +3,21 @@
     <%= render("shared/error_summary", instance: current_claim) if current_claim.errors.any? %>
 
     <%= form_for current_claim, url: claim_path do |form| %>
-      <h1 class="govuk-heading-xl">
-        <%= form.label :student_loan_repayment_amount, t("tslr.questions.student_loan_amount", claim_school_name: current_claim.claim_school_name)  %>
-      </h1>
+      <%= form_group_tag current_claim do %>
+        <h1 class="govuk-form-wrapper">
+          <%= form.label :student_loan_repayment_amount, t("tslr.questions.student_loan_amount", claim_school_name: current_claim.claim_school_name), class: "govuk-label govuk-label--xl" %>
+        </h1>
+
+        <span class="govuk-hint">Only include repayments made through your teaching wages. You can find this information on your payslips.</span>
+
+        <%= errors_tag current_claim, :student_loan_repayment_amount %>
 
-      <span class="govuk-hint">Only include repayments made through your teaching wages. You can find this information on your payslips.</span>
-      <div class="govuk-form-group govuk-currency-input">
```

Comment:
Is it ok that we've lost `govuk-currency-input`?

Reply:
Yes, but I've tied up the styles to match now as well.